pygmt/clib/session.py

```diff
@@ -5,6 +5,7 @@
 Uses ctypes to wrap most of the core functions from the C API.
 """
 import ctypes as ctp
+import pathlib
```

Comment:
This `import pathlib` is unused and can be removed.

```suggestion
```

Reply:
Done!